internal/file/getter.go

```diff
@@ -13,15 +13,16 @@ import (
 )
 
 var (
-	archiveExtensions = []string{
-		".tar",
-		".tar.gz",
-		".tgz",
-		".zip",
-	}
+	archiveExtensions   []string
```

Comment:
nit: It could help readers to understand how `archiveExtensions`'s value is set a bit more easily if we assign the value immediately here as we declare `archiveExtensions` instead of by using an `init()` function. (With this, instead of grabbing the values from `getter.Decompressors` in `init()`, we'd do this logic in a helper function.)

E.g.:

```go
var (
    archiveExtensions = decompressorNames()
    // ...
)
```